docs/resources/client/level.md

```diff
@@ -6,37 +6,39 @@ A level is a playable object in Geometry Dash, namely coming with data that expl
-| k63 | Third Coin Acquired      | **bool**                                                         | whether the third coin is acquired during verification                           |
-| k66 | Requested Stars          | **integer**                                                      | the requested stars during publication of the level                              |
-| k71 | Orb Reward Percentage    | **integer**                                                      | the percentage up until the orb reward has been granted                          |
-| k79 | Unlisted                 | **bool**                                                         | whether the level is to be marked as unlisted or not during publication          |
-| k80 | Seconds Spent Editing    | **integer**                                                      | the number of seconds spent editing the level                                    |
-| k83 | Difficulty               | **[Difficulty](enumerations.md)**                                | the level difficulty (including Demon ratings)                                   |
-| k84 | Level Folder             | **integer**                                                      | the folder in which the level belongs (0 represents no folder)                   |
+| Key | Name/Value                      | Type                                                             | Description                                                                      |
+|:----|:--------------------------------|:-----------------------------------------------------------------|:---------------------------------------------------------------------------------|
+| k1  | Level ID                        | **integer**                                                      | the id of the level                                                              |
+| k2  | Level Name                      | **string**                                                       | the name of the level                                                            |
+| k3  | Description                     | **string**                                                       | the level description, encoded in [base64](https://en.wikipedia.org/wiki/Base64) |
+| k4  | Inner Level String              | **[inner level string](level-components/inner-level-string.md)** | the inner level string, or the playable level                                    |
+| k5  | Creator                         | **[user](./user.md).name**                                       | the name of the level creator                                                    |
+| k6  | Creator's User ID               | **integer**                                                      | the user id of the level creator                                                 |
```

Comment:
Its type should be `user.id` to comply with the other user-related data. Follow k5's type notation